### ISSUE TITLE
Missing URI redirection configuration in Twitter provider

### ISSUE THREAD

**Issue by jesusprubio** [strapi/strapi#2346]
**Informations**
- **Node.js version**: 10.9.0
- **npm version**: 6.4.1
- **Strapi version**: 3.0.0-alpha.14.5
- **Database**: MongoDB v4.0.4
- **Operating system**: Fedora 29

**What is the current behavior?**
The UI shows a bad value for this parameter that the user has to add to the provider app setup: `.../auth/twitter/callback`. If the user add this one the process fails.
![bad](https://user-images.githubusercontent.com/2753855/48787281-c7981200-ece8-11e8-9321-429929918a7a.png)

**Steps to reproduce the problem**
Follow the documentation to setup Twitter provider.

**What is the expected behavior?**
The UI should should the correct value for this parameter: `.../connect/twitter/callback`
![good](https://user-images.githubusercontent.com/2753855/48787376-ed251b80-ece8-11e8-8923-39244c2fc288.png)

Moreover, this would be consistent with the URI provided by the rest of the providers.
![google](https://user-images.githubusercontent.com/2753855/48787428-04fc9f80-ece9-11e8-8418-f1701721eefe.png)
![fb](https://user-images.githubusercontent.com/2753855/48787434-0928bd00-ece9-11e8-8060-22174cfa5ab4.png)
![discord](https://user-images.githubusercontent.com/2753855/48787444-0ded7100-ece9-11e8-9291-f80104a66295.png)

**Suggested solutions**
To add a new case to cover Twitter in this switch statement: https://github.com/strapi/strapi/blob/17e2063943f5410fc014f6a95dfba30ec3ae8a93/packages/strapi-plugin-users-permissions/admin/src/components/PopUpForm/index.js#L45

I'm sending a PR.

**Comment by lauriejim** [comment]
Did you follow [the documentation guides](https://strapi.io/documentation/3.x.x/guides/authentication.html#providers) ?

**Comment by jesusprubio** [comment]
Yes, I've followed them, as I said in the issue report. By the way, they are not easy to follow at all because they are complex and they use a lot of React stuff which is not common for a lot of devs (like me) :(.

Moreover I think the problem is worse that we thought at first. I can't be sure because I'm in my first days with Strapi but I think this generated URI is used internally in any place (apart from the UI). Because, until I made these changes (even plugin rebuild), I couldn't get it working.

It's not a problem for my project because I have solve it in my code but I think it's not a good first contact with Strapi to try a provider and to see it's not working. At the end, this kind of out-of-the-box features are the biggest strengths of Strapi.
